src/runtime-rs/docs/Kata 3.0 Architecture.md

```diff
@@ -0,0 +1,96 @@
+## Overview
+In cloud-native scenarios, there is an increased demand for container startup speed, resource consumption, stability, and security. To achieve this, we would like to introduce you to a solid and  secure Rust version kata-runtime. We chose Rust because it is designed to prevent developers from accidentally introducing flaws in their code that can lead to buffer overflows, missing pointer checks, integer range errors, or other memory-related vulnerabilities. Besides, in contrast to Go, Rust makes a variety of design trade-offs in order to obtain the fastest feasible execution performance. 
+Also, we provide the following designs:
+
+- Turn key solution with builtin Dragonball Sandbox
+- Async io to reduce resource consumption
+- Extensible framework for multiple services, runtimes and hypervisors
+- Lifecycle management for sandbox and container associated resources
+## Design
+### Architecture
+![runD-new-架构.drawio.png](https://intranetproxy.alipay.com/skylark/lark/0/2022/png/12577/1651562957758-83619e84-06f4-49bf-92ca-f7bef38ad5db.png#clientId=u2af4c426-90b4-4&crop=0&crop=0&crop=1&crop=1&from=paste&height=731&id=u1b7b4365&margin=%5Bobject%20Object%5D&name=runD-new-%E6%9E%B6%E6%9E%84.drawio.png&originHeight=1462&originWidth=1442&originalType=binary&ratio=1&rotation=0&showTitle=false&size=205001&status=done&style=none&taskId=u15835d56-71cf-4433-82a9-594ac366d94&title=&width=721)
```

Comment:
Please delete Chinese characters.
And for images, you need to upload the source code too to let others edit it.

Reply:
fixed